From a2a45986f55925d18776aa572df65b050a5dd317 Mon Sep 17 00:00:00 2001
From: Xabier Marquiegui <xmarquiegui@ainguraiiot.com>
Date: Thu, 18 Feb 2021 12:18:54 +0100
Subject: [PATCH] Don't force gcc in the Makefile

This results in the host compiler being used in yocto builds instead of
the cross-compiler
---
 Makefile | 22 +++++++++++-----------
 1 file changed, 11 insertions(+), 11 deletions(-)

diff --git a/Makefile b/Makefile
index e69cdfe..4b856f7 100644
--- a/Makefile
+++ b/Makefile
@@ -143,7 +143,7 @@ obj-$(CONFIG_SPL_BUILD) += xor.o
 # ******************************
 else ifneq ($(TOOL),)
 
-CC       = gcc
+#CC       = gcc
 RM       = @rm -rf
 ECHO     = @echo
 
@@ -245,7 +245,7 @@ clean:
 # MARVELL A3700 SUPPORT
 # *********************
 else ifneq ($(findstring a3700,$(PLATFORM)),)
-CC       = gcc
+#CC       = gcc
 RM       = @rm -rf
 ECHO     = @echo
 
@@ -311,15 +311,15 @@ clean:
 # MARVELL ATF SUPPORT
 # *******************
 else
-CROSS    = $(CROSS_COMPILE)
-
-LD       = $(CROSS)ld
-CC       = $(CROSS)gcc
-AS       = $(CROSS)gcc
-AR       = $(CROSS)ar
-OBJCOPY  = $(CROSS)objcopy
-OBJDUMP  = $(CROSS)objdump
-STRIP    = $(CROSS)strip
+#CROSS    = $(CROSS_COMPILE)
+
+#LD       = $(CROSS)ld
+#CC       = $(CROSS)gcc
+#AS       = $(CROSS)gcc
+#AR       = $(CROSS)ar
+#OBJCOPY  = $(CROSS)objcopy
+#OBJDUMP  = $(CROSS)objdump
+#STRIP    = $(CROSS)strip
 
 RM       = @rm -rf
 MKDIR    = @mkdir -p
